source/mainnet/net/installation/previous-node-downloads.rst

```diff
@@ -33,6 +35,8 @@ Windows - Mainnet and Testnet
 macOS - Mainnet and Testnet
 ===========================
 
+`macOS 5.0.6 <https://distribution.concordium.software/macos/signed/concordium-node-5.0.6-0.pkg>`_
```

Comment:
I assume this should be replaced with 5.0.7

Reply:
Fixed.